docs/source/guides/helm.rst

```diff
@@ -0,0 +1,202 @@
+Using Helm to install YataiService
+=============================================
+
+*Helm* is a tool that streamlines installing and managing K8s applications. It lets us easily configure a set of K8s resources that make up our application -- YataiService in this case. Helm enables us to
```

Comment:
It's better to avoid the first person ("we/us/our") because the reader can become confused. It's not clear who "us" is in this context.